addons/mail/security/ir.model.access.csv

```diff
@@ -66,3 +66,5 @@ access_res_users_settings_volumes_user,res.users.settings.volumes,model_res_user
 access_mail_template_reset,access.mail.template.reset,model_mail_template_reset,mail.group_mail_template_editor,1,1,1,1
 ir_actions_report_access_user,ir.actions.report.access.user,base.model_ir_actions_report,base.group_user,1,0,0,0
 access_mail_link_preview_admin,mail.link.preview.admin,model_mail_link_preview,base.group_erp_manager,1,1,1,1
+access_discuss_voice_attachment_user,discuss.voice.attachment.user,model_discuss_voice_attachment,base.group_user,1,1,1,1

```

Comment:
@maki-odoo you show JS code but Xavier is asking for record rules, to limit the access server side.

Reply:
You are right, what I am trying to do is to keep the logic as close to attachments as possible, so it can be used as meta data for the attachments, and there is no rules defined for the attachments.